### PR TITLE
Add APM extension books

### PR DESCRIPTION
**Pull request by bmorelli25**
Adds new APM extension books. Why? See https://github.com/elastic/observability-docs/issues/2207.

### CODE REVIEW

File: conf.yaml [hunk 1]
```diff
@@ -1440,6 +1440,32 @@ contents:
                         repo:   apm-agent-rum-js
                         path:   CHANGELOG.asciidoc
                         exclude_branches:   [ 3.x, 2.x, 1.x, 0.x ]
+              - title:      APM AWS Lambda extension
```
Comment (dedemorton) [inline conf.yaml:1443]: I don't have strong feelings about this, but I think we use title caps for all the other book names.

**Review by dedemorton — APPROVED** [review]
LGTM by eyeballing it (assuming the build passes).

File: conf.yaml (hunk 1 above)
Comment (bmorelli25) [reply conf.yaml:1443]: Ah. We do. Thanks.

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Pull request merged**